tslint.json

```diff
@@ -1,6 +1,24 @@
 {
   "extends": "tslint-config-standard",
   "rules": {
+    "member-ordering": [
+      true,
+      {
+        "order": [
```

Comment:
why are we overriding the default here?

Reply:
This is let us have private methods mixed with other methods so that we can have private methods straight underneath the method that uses it.

I did not find other way to remove rule describing order of private methods other then overriding whole configuration node.

Reply:
sounds like too many methods if that's a problem?
I would prefer private methods separate.
I recommend reverting this, but open to hearing others opinions

Reply:
I don't mind having private methods mixed between public ones as long as they maintain a logical order -- e.g. something Damian mentioned where the private method is straight underneath the public method that uses it.